{{ cookiecutter.dir_name }}/src/android/PythonActivity.java

```diff
@@ -3,9 +3,10 @@
 import android.app.Activity;
 import android.os.Bundle;
 import android.util.Log;
+import android.support.v7.app.AppCompatActivity;
 
 
-public class PythonActivity extends Activity {
+public class PythonActivity extends AppCompatActivity {
```

Comment:
I take it this change is to support the theme stuff? Are there any other features we gain as a result?